CMakeLists.txt

```diff
@@ -274,6 +276,7 @@ target_include_directories(${PDAL_BASE_LIB_NAME}
         ${NLOHMANN_INCLUDE_DIR}
         ${GDAL_INCLUDE_DIR}
         ${LASZIP_INCLUDE_DIR}
+        ${draco_INCLUDE_DIR}
```

Comment:
We don't want to leak this into our public interface. The target_include_directories should be set for every Draco target that needs this to `PRIVATE`.